docs/src/main/sphinx/connector/hive.rst

```diff
@@ -484,7 +484,15 @@ with Parquet files performed by the Hive connector.
         definition. The equivalent catalog session property is
         ``parquet_use_column_names``.
       - ``true``
-
+    * - ``parquet.writer.page-size``
```

Comment:
You could define these separately as a document fragment and import them here to avoid the duplication

Reply:
In `delta-lake.rst` are defined the session properties for the connector.
In `hive.rst` are defined configuration properties for the connector.

They don't really overlap.
However, yes, it would make sense to expose those connector settings for Delta Lake connector as well. Follow-up PR